### PR TITLE
Prerequisite changes to fix #3695

### LINKED ISSUE #3695: [GO] Handle global variable and constant
Global variable and constant is declared here: https://github.com/apm-dev/go-user-rest-api/blob/main/datasources/mysql/users_db/users_db.go

global variable is used here: https://github.com/apm-dev/go-user-rest-api/blob/main/domain/users/user_dao.go
(stmt, err := users_db.Client.Prepare(queryGetAll))

End outcome to verify: MethodFullName property for below call nodes would be get resolved:

cpg.call("Prepare").l

### PR DESCRIPTION
**Pull request by pandurangpatil**
Prerequisite changes to fix #3695 issue.

Handling done to set the member node under given TypeDecl parent.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -64,7 +64,7 @@ ThisBuild / compile / javacOptions ++= Seq(
 ThisBuild / scalacOptions ++= Seq(
   "-deprecation", // Emit warning and location for usages of deprecated APIs.
   "--release",
-  "11",
+  "11"
```
Comment (DavidBakerEffendi) [inline build.sbt:67]: To avoid unrelated change
```suggestion
  "11",
```

Comment (pandurangpatil) [reply build.sbt:67]: This is formattee change. It has removed the unwanted last `,`

File: joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/passes/base/AstLinkerPass.scala [hunk 2]
```diff
@@ -16,6 +16,9 @@ class AstLinkerPass(cpg: Cpg) extends CpgPass(cpg) with LinkingUtil {
     cpg.typeDecl.whereNot(_.astParent).foreach { typeDecl =>
       addAstParent(typeDecl, typeDecl.fullName, typeDecl.astParentType, typeDecl.astParentFullName, dstGraph)
     }
+    cpg.member.whereNot(_.astParent).foreach { member =>
+      addAstParent(member, member.typeFullName, member.astParentType, member.astParentFullName, dstGraph)
```
Comment (DavidBakerEffendi) [inline joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/passes/base/AstLinkerPass.scala:20]: `member.typeFullName` is not this member's full name. Members don't really have full names, though, but this is only used to log errors. Perhaps you could do something like 
```suggestion
      addAstParent(member, s"${member.astParentFullName}.${member.name}", member.astParentType, member.astParentFullName, dstGraph)
```

**Review by DavidBakerEffendi — APPROVED** [review]
